pennylane/transforms/get_unitary_matrix.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A transform to obtain the matrix representation of a quantum circuit.
+"""
+from functools import wraps
+import numpy as np
+from pennylane.wires import Wires
+import pennylane as qml
+
+
+def get_unitary_matrix(circuit, wire_order=None):
+    r"""Construct the matrix representation of a quantum circuit.
+
+    Args:
+        circuit (.QNode, .QuantumTape, or Callable): A quantum node, tape, or function that applies quantum operations.
```

Comment:
@ingstra Sphinx is failing with this warning:
```
/github/workspace/pennylane/transforms/get_unitary_matrix.py:docstring of pennylane.transforms.get_unitary_matrix.get_unitary_matrix:: WARNING: more than one target found for cross-reference 'QNode': pennylane.QNode, pennylane.beta.QNode
```

This is unrelated to your PR, and due to a recent change in `master`! This change will fix it.
```suggestion
        circuit (pennylane.QNode, .QuantumTape, or Callable): A quantum node, tape, or function that applies quantum operations.
```